setup.md

```diff
@@ -46,20 +46,38 @@ inline error checking, and visual debugging features.
 
 To install and run Flutter, your development environment must meet these minimum requirements:
 
-* **Operating Systems**: Mac or Linux (64-bit). For Windows instructions (beta), see [Flutter on Windows](https://github.com/flutter/flutter/wiki/Flutter-on-Windows).
+* **Operating Systems**: Mac or Linux or Windows. (64-bit)
 * **Disk Space**: 700 MB on Mac, 600 MB on Linux, 400 MB on Windows (does not include disk space for Xcode or Android Studio).
-* **Tools**: Flutter depends on these command-line tools being available on your environment.
-  * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
+* **Tools**: Flutter depends on these command-line tools being available in your environment.
+  * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`(Mac and Linux), 
+  * Git with command-line support, PowerShell (Windows)
```

Comment:
Actually, I am not sure I understand the benefit of calling out a bunch of tools that come with the operating system (not just for Windows, same for Mac and Linux)?